src/platform/site-wide/sass/modules/_m-responsive-table.scss

```diff
@@ -15,3 +15,12 @@
     }
   }
 }
+.mobile-value p {
+  margin-left: 2px;
+}
+.mobile-value strong {
+  font-weight: normal;
+}
+.mobile-header {
+  margin-bottom: inherit;
+}
```

Comment:
It might be nice to add a newline at the end of this file and in-between the CSS selectors, similar to [above](https://github.com/department-of-veterans-affairs/vets-website/pull/14746/files#diff-2a97d6d6cf84e5724fa1f2a54ede937aaaa3d7468d02ac5b41a5c97c143a99c6R8) 🙂

Reply:
Why can't these be replaced with VADS utility classes? `margin-bottom: inherit;` is also a little confusing, since we control the HTML.

Reply:
it turns out this class wasn't being used, and i was already using `vads-u-margin-bottom` for the appropriate `p`

Reply:
Oh okay, cool, so we can remove the rules from this SCSS file then?